components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationMgtDBQueries.java

```diff
@@ -274,7 +274,10 @@ public class ApplicationMgtDBQueries {
     public static final String REMOVE_UM_PERMISSIONS = "DELETE FROM UM_PERMISSION WHERE UM_ID = ?";
     public static final String REMOVE_UM_ROLE_PERMISSION = "DELETE FROM UM_ROLE_PERMISSION WHERE UM_PERMISSION_ID = ?";
 
-    // DELETE query - Oauth
+    // DELETE query - OAuth.
+    @Deprecated
+    // Use the REMOVE_OAUTH_APPLICATION_WITH_TENANT query in
+    // org.wso2.carbon.identity.application.mgt.dao.impl.ApplicationMgtDBQueries class.
```

Comment:
let's follow standard javaDoc notation
```suggestion
    /**
 * @deprecated Use the REMOVE_OAUTH_APPLICATION_WITH_TENANT query in
 * {@link org.wso2.carbon.identity.application.mgt.dao.impl.ApplicationMgtDBQueries} class.
 */
```